Source/Extensions/Blazorise.RichTextEdit/JSRichTextEditModule.cs

```diff
@@ -60,15 +63,15 @@ public async ValueTask<IAsyncDisposable> Initialize( RichTextEdit richTextEdit )
     {
         var dotNetRef = DotNetObjectReference.Create( richTextEdit );
 
-        await InvokeSafeVoidAsync( "initialize",
-            dotNetRef,
-            richTextEdit.ElementRef,
-            richTextEdit.ElementId,
-            richTextEdit.ReadOnly,
-            richTextEdit.PlaceHolder,
-            richTextEdit.Theme == RichTextEditTheme.Snow ? "snow" : "bubble",
-            richTextEdit.SubmitOnEnter,
-            richTextEdit.ConfigureQuillJsMethod );
+        await InvokeSafeVoidAsync( "initialize", dotNetRef, richTextEdit.ElementRef, richTextEdit.ElementId, new RichTextEditJSOptions
+        {
+            ReadOnly = richTextEdit.ReadOnly,
+            Placeholder = richTextEdit.PlaceHolder,
+            Theme = richTextEdit.Theme == RichTextEditTheme.Snow ? "snow" : "bubble",
+            SubmitOnEnter = richTextEdit.SubmitOnEnter,
+            ConfigureQuillJsMethod = richTextEdit.ConfigureQuillJsMethod,
+            UseTables = options.UseTables,
```

Comment:
Not worth having it as a RichTextEdit parameter?

Reply:
The table is added as any other RTE action by adding a button toolbar. 

```
<RichTextEditToolbarButton Action="RichTextEditAction.Table" />
```

So, there is no need for extra parameters.